lib/poteto/config.rb

```diff
@@ -4,37 +4,43 @@
 
 module Poteto
   class ReviewerConfig
-    attr_reader :exclude, :reviewer
-    attr_accessor :commit_id
+    attr_reader :exclude, :reviewer, :repo
+    attr_accessor :commit_id, :pr_id
 
-    def initialize(reviewer, reviewer_config)
+    def initialize(reviewer, reviewer_config, config)
       @reviewer = reviewer.to_sym
       @exclude = reviewer_config[:exclude] || []
+      @repo = reviewer_config.fetch(:repository)
+      @config = config
+    end
+
+    def access_token
+      @config.access_token
     end
   end
 
   class Config
-    attr_reader :file_path, :reviewers, :repo, :config, :commit_id
-    attr_accessor :pr_id, :access_token
+    attr_reader :file_path, :reviewers, :repo, :config, :commit_id, :pr_id
+    attr_accessor :access_token
 
     DEFAULT_CONFIG_FILE = '.poteto.yaml'
     DEFAULT_CONFIGS = {
       reviewers: {
         rubocop: {
           exclude: %w[spec/ test/]
-        }
+        },
+        repository: "hjadskhald"
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.